test/PerformanceTests/SerializationBaselineTests/SerializationBaselineTests.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>

```

Comment:
Any specific reason to not use net5, the latest release? AGS is also on net5 FWIW.

Reply:
Added .net5